packages/browser/src/plugins/segmentio/normalize.ts

```diff
@@ -131,7 +131,8 @@ export function normalize(
   const ctx = json.context
 
   // This guard against missing ctx.page should not be neccessary, since context.page is always defined
-  const query: string = ctx.page?.search || ''
+  const query: string =
+    typeof ctx.page?.search === 'string' ? ctx.page?.search : ''
```

Comment:
This line needs a unit tests, correct? Page calls should be able to take search as an object without throwing.